Brightspace/insights-adoption-dashboard
# US135379: if end day is today, send a null instead (fix regression)

My earlier changes started sending a date when the end date is today's date instead of `null`. The `null` is needed for cases where there may be server data for "tomorrow", e.g. the server is in Australia but my browser is in Ontario.

testing (e2e)
- load with no query params: expected date range appears, and end date is sent to server as null
- change start date: expected dates appear in url (start and end), and end date is sent to server as null
  - reload: loads same
- change end date: appears in url, both start and end dates are sent to the server
  - reload: loads same
